spec/models/manageiq/providers/redhat/infra_manager/provision/disk_spec.rb

```diff
@@ -164,4 +164,61 @@
       @task.poll_add_disks_complete
     end
   end
+
+  context "#find_storage!" do
+    let(:storage_name) { "test_storage" }
+    let(:storage_id) { 123 }
+
+    context "when storage exists" do
+      let(:storage) { FactoryGirl.create(:storage, :name => storage_name, :id => storage_id) }
+
+      it "searches for a storage by its ID" do
```

Comment:
I would change the "searcher" to "finds" here and in the next examples because when I read this I thought you will have something like expect(Storage).to receive(:find)